common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -470,6 +470,74 @@ static bool get_scan_results_from_nl_msg(sChannelScanResults &results, struct nl
     return true;
 }
 
+static std::shared_ptr<char> generate_client_assoc_event(const std::string &event, int vap_id,
+                                                         bool radio_5G)
```

Comment:
`radio_5G` is not used please remove it from the signature

Reply:
Removed